app/views/organisations/_videos.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="brand--<%= @organisation.brand %> brand__border-color organisation__brand-border-top organisation__section-wrap organisation__margin-bottom" id="videos">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/heading", {
+          text: t("organisations.videos"),
+          padding: true,
+          lang: "en"
+      } %>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/govspeak" do %>
+        <a href="<%= @documents.promotional_video_feature_youtube_url %>" class="govuk-link brand__color"><%= @documents.promotional_video_feature_youtube_url %></a>
+      <% end %>
+      <p class="govuk-heading-s govuk-!-margin-top-1"><%= @documents.promotional_video_feature_title %></p>
```

Comment:
Ideally the govuk margin override classes should only be used where overrides are necessary, but I appreciate that this work is time sensitive. Might be good to come back to these use of these classes later and write some CSS to support the layout here.

Reply:
@andysellick I didn't know this - thanks for letting me know!